config/locales/user_mailer/en.yml

```diff
@@ -11,26 +11,34 @@ en:
       'yes': 'yes'
     email_changed:
       help: >
-        If you did not change your email address, please contact the %{app}
-        Customer Contact Center at %{link}.
+        If you did not change your email address, please immediately call the
+        %{app} customer contact center at 1-844-USA-GOV1 (1-844-872-4681)
+        between 8:00 AM and 8:00 PM Eastern Time, Monday through Friday, except
+        federal holidays.
       intro: >
         You have asked %{app} to change the email address currently associated
         with your account.
     password_changed:
       help: >
-        If you did not mean to change your password, please immediately contact
-        the %{app} customer contact center via web form at %{link}.
+        If you did not mean to change your password, please immediately call the
+        %{app} customer contact center at 1-844-USA-GOV1 (1-844-872-4681)
+        between 8:00 AM and 8:00 PM Eastern Time, Monday through Friday, except
+        federal holidays.
       intro: You have changed your password for your %{app} account.
     phone_changed:
       help: >
-        If you did not mean to change your phone, please immediately contact
-        the %{app} customer contact center via web form at %{link}.
+        If you did not mean to change your phone, please immediately call the
+        %{app} customer contact center at 1-844-USA-GOV1 (1-844-872-4681)
+        between 8:00 AM and 8:00 PM Eastern Time, Monday through Friday, except
+        federal holidays.
       intro: You have changed your phone number for your %{app} account.
       subject: Your phone number has been changed
     signup_with_your_email:
       help: >
         If you did not request a new account or suspect an error, please
-        immediately contact the %{app} customer contact center via web form at %{link}.
+        immediately call the %{app} customer contact center at 1-844-USA-GOV1
+        (1-844-872-4681) between 8:00 AM and 8:00 PM Eastern Time, Monday
+        through Friday, except federal holidays.
```

Comment:
Based on experience at my last company, when translations are involved, it's easier to duplicate the text like this because in other languages the sentence structure may not make it easy to combine fragments like it is in English